modules/photo/test/test_hdr.cpp

```diff
@@ -249,4 +249,22 @@ TEST(Photo_CalibrateRobertson, regression)
     checkEqual(expected, response, 1e-1f, "CalibrateRobertson");
 }
 
+TEST(Photo_CalibrateRobertson, bug_18180)
+{
+    vector<Mat> images;
+    vector<cv::String> fn;
+    glob(string(cvtest::TS::ptr()->get_data_path()) + "hdr/exposures/bug_18180/*.jpg", fn, false);
+    for (auto & i : fn)
+        images.push_back(imread(i));
```

Comment:
Please avoid using `glob()` in tests. Use fixed input instead.

Reply:
Got it!